src/main/java/io/codelex/polymorphism/practice/exercise1/DragRace.java

```diff
@@ -13,6 +16,94 @@
 public class DragRace {
 
     public static void main(String[] args) {
+        Audi audi = new Audi();
+        Bmw bmw = new Bmw();
+        Lexus lexus = new Lexus();
+        Mazda mazda = new Mazda();
+        Tesla tesla = new Tesla();
+        Toyota toyota = new Toyota();
+
+        List<Object> cars = Arrays.asList(audi, bmw, lexus, mazda, tesla, toyota);
+
+        speedUpCars(cars);
+        checkFastestCar(cars);
+    }
+
+    private static void speedUpCars(List<Object> cars) {
+        for (int i = 0; i < 10; i++) {
+            for (Object car : cars) {
+                if (car instanceof Audi) {
```

Comment:
šis garais noteikti nav nepieciešams, visas ir mašīnas un visas taisa `speedUp()` neatkarīgi no tā vai tas ir bmw vai audi